Rakefile

```diff
@@ -169,11 +179,26 @@ REGEN_METADATA = {
         ns: 'Azure::ARM::Compute',
         version: version
     },
+    azure_mgmt_consumption: {
+        spec_uri: 'https://raw.githubusercontent.com/Azure/azure-rest-api-specs/current/specification/consumption/resource-manager/readme.md',
+        ns: 'Azure::ARM::Consumption',
+        version: version
+    },
+    azure_mgmt_container_instance: {
+        spec_uri: 'https://raw.githubusercontent.com/Azure/azure-rest-api-specs/current/specification/containerinstance/resource-manager/readme.md',
+        ns: 'Azure::ARM::ContainerInstance',
+        version: version
+    },
     azure_mgmt_container_registry: {
         spec_uri: 'https://raw.githubusercontent.com/Azure/azure-rest-api-specs/current/specification/containerregistry/resource-manager/readme.md',
         ns: 'Azure::ARM::ContainerRegistry',
         version: version
     },
+    azure_mgmt_customer_insights: {
+        spec_uri: 'https://raw.githubusercontent.com/Azure/azure-rest-api-specs/current/specification/customer-insights/resource-manager/readme.md',
+        ns: 'Azure::ARM::ContainerInsights',
```

Comment:
Should `Azure::ARM::ContainerInsights` be `Azure::ARM::CustomerInsights` ?

Reply:
Thanks for catching it, this one I addressed with a later commit, so it should be good in latest.